homeassistant/core.py

```diff
@@ -2075,20 +2152,27 @@ def async_set(
             entity_id,
             new_state,
             attributes,
-            last_changed,
-            now,
-            context,
-            old_state is None,
-            state_info,
+            last_changed=last_changed,
+            last_reported=now,
+            last_updated=now,
+            context=context,
+            validate_entity_id=old_state is None,
+            state_info=state_info,
```

Comment:
This one bumps up a bit with the change to kwargs on the profile since the arg unpacking can't take the fast path